### ISSUE TITLE
Handling 'derivative' or 'identical' products

### ISSUE THREAD

**Issue by mbergman42** [mbergman42/USCTM-90-Day#7]
The following is input from the HDMI® Licensing Administrator, Inc. (HDMI LA) by way of Brad Bramy:  
   
For products that are identical or for a core product with derivatives, there is can be literally hundreds of derivatives under the top unit. Consumers will be looking for a model number that is what they saw when purchasing or searching.  

A possible hierarchy is, 
   
Manufacturer of certified base model "ODM"  _<-- required for database_
   / Certified base model "BASE" from ODM (may not actually be a salable product)  _<-- required for database_
      / Marketed brand name "BRAND" _<--required for label_
         / Marketed product name "MODEL"  _<-- required for label_

**Comment by mbergman42** [comment]
Some interesting use cases:

- ODM has a BASE model that they sell to multiple BRANDs under various MODELs. Fields/parameters/URLs in the Registry need to be allocated to the ODM or BRAND as appropriate. Technical issues (security contact, e.g.) should go to the ODM. Consumer-facing details (support page URL, e.g.) should go to the BRAND. 
- ODM has multiple customers, none of whom can know each other. ODM needs to be able to register multiple BASE models to keep from identifying what they are selling to their BRAND customers. 
- BRAND has a MODEL number that applies to multiple actual product models. E.g., TV maker has a model number that doesn't indicate screen size, bezel color, other options. The TV maker should be able to identify a BASE and MODEL number that doesn't have to deal with these details.

**Comment by mbergman42** [comment]
- ODM has a BASE model that they sell to multiple BRANDs under various MODELs. Three cases: 

1. The ODM submits the BASE model to the CyberLAB for testing; 
2. The BRAND submits the MODEL to the CyberLAB for testing;
3. The ODM submits the BRAND's MODEL to the CyberLAB for testing. 

As follow-up, who is the Applicant to the CLA? Should be the BRAND but the BRAND may assign Applicant status and tasks to their ODM.  

R14 WG7 and R14 WG9 should be able to handle these situations.

**Comment by jordan2175** [comment]
Yeah, these are the issues. From my standpoint I guess it all comes down to who is the customer going to contact, or should they contact, if there is a problem. If they are white labeled, this should be the Brand that is selling them and has their logo on it.
